seed/csharp-sdk/streaming-parameter/src/SeedStreaming.Test/Unit/MockServer/BaseMockServerTest.cs

```diff
@@ -1,39 +0,0 @@
-using NUnit.Framework;
```

Comment:
Note that we can remove this file since there aren't any mock tests generated for this test definition.